Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/ComboBoxInput.swift

```diff
@@ -79,29 +74,34 @@ struct ComboBoxInput: View {
     }
     
     var body: some View {
-        HStack {
-            Text(displayedValue)
-                .accessibilityIdentifier("\(element.label) Combo Box Value")
-                .frame(maxWidth: .infinity, alignment: .leading)
-                .foregroundStyle(!selectedValue.isNoValue ? .primary : .secondary)
-            if let _ = selectedValue.codedValue, !isRequired {
-                // Only show clear button if we have a value
-                // and we're not required. (i.e., Don't show clear if
-                // the field is required.)
-                XButton(.clear) {
-                    embeddedFeatureFormViewModel.focusedElement = element
-                    defer { embeddedFeatureFormViewModel.focusedElement = nil }
-                    updateValueAndEvaluateExpressions(nil)
+        Button {
+            embeddedFeatureFormViewModel.focusedElement = element
+            isPresented = true
+        } label: {
+            HStack {
+                Text(displayedValue)
+                    .accessibilityIdentifier("\(element.label) Combo Box Value")
+                Spacer()
+                if let _ = selectedValue.codedValue, !isRequired {
+                    // Only show clear button if we have a value
+                    // and we're not required. (i.e., Don't show clear if
+                    // the field is required.)
+                    XButton(.clear) {
+                        embeddedFeatureFormViewModel.focusedElement = element
+                        defer { embeddedFeatureFormViewModel.focusedElement = nil }
+                        updateValueAndEvaluateExpressions(nil)
+                    }
+                    .accessibilityIdentifier("\(element.label) Clear Button")
+                    .buttonStyle(.plain)
+                } else {
+                    // Otherwise, always show chevron.
+                    Image(systemName: "chevron.right")
+                        .accessibilityIdentifier("\(element.label) Options Button")
                 }
-                .accessibilityIdentifier("\(element.label) Clear Button")
-            } else {
-                // Otherwise, always show chevron.
-                Image(systemName: "chevron.right")
-                    .accessibilityIdentifier("\(element.label) Options Button")
-                    .foregroundStyle(.secondary)
             }
+            .foregroundStyle(!selectedValue.isNoValue ? .primary : .secondary)
```

Comment:
In my experience, "conveniences" like `isNoValue` end up being no convenience at all. To understand why, consider what this code would look like with and without the convenience:

```swift
// With
.foregroundStyle(!selectedValue.isNoValue ? .primary : .secondary)
// Without
.foregroundStyle(selectedValue != .noValue ? .primary : .secondary)
```

The code is _nearly identical_. Consider removing the `is<something>` properties.

Speaking of `ComboBoxValue`, including `Value` in each case is repeating type information. I strongly suggest removing the `Value` suffix to conform to the Swift API design guidelines. Along those lines, the `noValue` case, as far as I can tell, would be better represented by `nil`:

```swift
@State private var selectedValue: ComboBoxValue?
```